examples/complex_numbers.js

```diff
@@ -44,6 +44,36 @@ print(c);                      // 1 + i
 var d = math.complex(3, 4);
 console.log(d.abs(), d.arg());      // radius = 5, phi = 0.9272952180016122
 
+// comparisions between two complex numbers (only considers real components)
+console.log('\ncomparisions between two complex numbers (only considers real components)');
```

Comment:
I think the comparison examples here are too comprehensive. Can you replace this with say two or the examples and a few lines of comments which explaining the behavior?